### PR TITLE
Update to gordon ergo-infinity

### PR DESCRIPTION
**Pull request by DanielGGordon**
## Description

    * Added combos
    * Added layers for windows snapping
    * Extended tapdance for more than 2x holds/taps

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

**Comment by DanielGGordon** [comment]
ah should not have done this to master branch I take it...

File: keyboards/input_club/ergodox_infinity/keymaps/gordon/keymap.c [hunk 1]
```diff
@@ -4,20 +4,67 @@
 #include "version.h"
 #include "process_keycode/process_tap_dance.h"
 #include "gordon.h"
-
-#include "keymap_german.h"
-
-#include "keymap_nordic.h"
-
+#include "ergodox_infinity.h"
```
Comment (drashna) [inline keyboards/input_club/ergodox_infinity/keymaps/gordon/keymap.c:7]: The QMK_KEYBOARD_H include is already including this file.
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
> ah should not have done this to master branch I take it...

To master is fine.  from your master makes things more difficult once it's merged, for you.

**Review by drashna — APPROVED** [review]

File: keyboards/input_club/ergodox_infinity/keymaps/gordon/rules.mk [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
 
+# two lines below are to re-enable the LED screens
+ST7565_ENABLE = no
+VISUALIZER = yes
+VISUALIZER_ENABLE = yes
+LCD_ENABLE = yes
+LCD_BACKLIGHT_ENABLE = yes
+LCD_DRIVER = st7565
+LCD_WIDTH = 128
+LCD_HEIGHT = 32
```
Comment (fauxpark) [inline keyboards/input_club/ergodox_infinity/keymaps/gordon/rules.mk:13]:
```suggestion
```
Visualizer has been completely removed now.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**